### PR TITLE
Helm publish chart action and 0.12.0 release

### PR DESCRIPTION
**Pull request by cevian**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  push:
+    tags: ['*']
+
+env:
+  helm-version: v3.8.1
+
+jobs:
+  publish-chart:
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/')
```
Comment (feikesteenbergen) [inline .github/workflows/release.yml:13]: Nit, not sure if needed? as we already filter on tags on line 5?
```suggestion
```

**Review by feikesteenbergen — APPROVED** [review]
LGTM, I guess we can add multinode once we are happy with how this works?

File: .github/workflows/release.yml (hunk 1 above)
Comment (cevian) [reply .github/workflows/release.yml:13]: I would like to keep it as a safety check if we ever change line 5

**Comment by cevian** [comment]
@feikesteenbergen yes we can add multinode later

**Pull request merged**